Source/Krypton Components/Krypton.Toolkit/Palette Base/PaletteDefinitions.cs

```diff
@@ -1068,9 +1068,9 @@ public interface IPalette
 
 		#region BaseFontSize
```

Comment:
If you undoing 703, then there other changes that need to be undone (i.e. Changelog.md)

Reply:
Forgot about that, thanks for reminding me :)